4_cv_basics/1_image_representation/Makefile

```diff
@@ -5,9 +5,9 @@ CC = g++
 PROJECT = image_representation		
 
 # Here Include specifies which directories can be added in the include path of the main.cpp
-CFLAGS = -std=c++11 $(shell pkg-config --cflags opencv4) -Iinclude/		
+CFLAGS = -std=c++11 $(shell pkg-config --cflags opencv4) $(shell pkg-config --cflags sdl2)		
```

Comment:
were cflags necessary for sdl2?
didn't it work without them?

Reply:
no it's not working without --cflags sdl2

Reply:
Then why did it work before in linux? As there was no cflag before

Reply:
It does not work because the directory structure is slightly different in macOS, therefore giving an error for `SDL2/SDL.h`

Reply:
> It does not work because the directory structure is slightly different in macOS, therefore giving an error for `SDL2/SDL.h`

Sure, got it!

Reply:
I think then this change is good to go